File: it-jobs-be/user-service/src/main/java/com/techjobs/model/UserRole.java
package com.techjobs.model;

public enum UserRole {

    ROLE_ADMIN("Admin"),

    ROLE_APPLICANT("Applicant"),

    ROLE_COMPANY("Company");

    private final String label;

    // Constructor
    UserRole(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }

    @Override
    public String toString() {
        return label;
    }
}

File: it-jobs-fe/src/constants/userRole.js
import PersonIcon from '@mui/icons-material/Person';
import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
import BusinessIcon from '@mui/icons-material/Business';

export const USER_ROLE_OPTIONS = [
  { value: 'ROLE_ADMIN', label: 'Admin', icon: <PersonIcon fontSize="small" sx={{ mr: 0.5 }} /> },
  { value: 'ROLE_APPLICANT', label: 'Applicant', icon: <PersonOutlineIcon fontSize="small" sx={{ mr: 0.5 }} /> },
  { value: 'ROLE_COMPANY', label: 'Company', icon: <BusinessIcon fontSize="small" sx={{ mr: 0.5 }} /> },
];